src/protocols/ethereum/config.ts

```diff
@@ -10,6 +10,46 @@ export const ETH_COIN_PRECISION = 18; // Amount of decimals
 export const ETH_COINGECKO_COIN_ID = 'ethereum';
 export const ETH_GAS_LIMIT = 21000;
 
+export const ERC20_ABI = [
```

Comment:
```suggestion
export const ERC20_ABI: AbiItem[] = [
```

The `AbiItem` type is probably available in the 'web3-utils' package.